visma/functions/constant.py

```diff
@@ -1,10 +1,11 @@
 import math
-from visma.functions.structure import Function
+from visma.functions.structure import Function, Expression
 from visma.functions.variable import Variable
+from visma.functions.exponential import Exponential
 
-############
-#   Constant #
-############
+#############
+# Constant  #
+#############
 
 
```

Comment:
You can update the `__init__` in `Constant` class.  It can be something similar to this.
```python
def __init__(self, value=None, power=1, coefficient=1):
    super().__init__()
    self.coefficient = coefficient
    self.power = power
    self.type = 'Constant'
    if value is not None:
        self.value = value
    if self.power != 1 or self.coefficient != 1:
        self.value = self.coefficient * (self.value**self.power)
        self.coefficient = 1
        self.power = 1
```
It will make it easier to initialize Constants. Update in test_functionsn accordingly.